tests/core/full_node/test_conditions.py

```diff
@@ -87,7 +87,7 @@ async def check_spend_bundle_validity(
         )
         newest_block = additional_blocks[-1]
 
-        received_block_result, err, fork_height = await blockchain.receive_block(newest_block)
+        received_block_result, err, fork_height, coin_changes = await blockchain.receive_block(newest_block)
```

Comment:
this test doesn't seem to verify that `coin_changes` is correct

Reply:
`receive_block()` is called from many places in the tests. I didn't add verification to those, but had to add `, coin_changes` or it would fail to unpack the return value.